PCampReview.py

```diff
@@ -158,15 +155,80 @@ def infoPopup(self,message):
     messageBox = qt.QMessageBox()
     messageBox.information(None, 'Slicer mpMRI review', message)
 
+  def setupIcons(self):
```

Comment:
We can probably just remove the text and have more intuitive buttons, with the name of the step in tooltip on mouseover.